### PR TITLE
Only color paths, not groups

### PR DESCRIPTION
**Pull request by josephspens**
By filling in group tags with their child paths, opacities would be applied to both paths and their parents, compounding the visual effect.

### CODE REVIEW

File: src/ticketmap/index.tsx [hunk 1]
```diff
@@ -444,7 +444,10 @@ export default class TicketMap extends Component<Props, State> {
 
   getAllPaths = (id) =>
     Array.from(this.mapRootRef.querySelectorAll(`[data-section-id${id ? `="${id}"` : ''}]`))
```
Comment (mayavera) [inline src/ticketmap/index.tsx]:
```suggestion
    Array.from(this.mapRootRef.querySelectorAll(`path[data-section-id${id ? `="${id}"` : ''}]`))
```

Comment (josephspens) [reply src/ticketmap/index.tsx]: We don't want to do this because it would exclude groups without paths, or elements other than paths

Comment (mayavera) [reply src/ticketmap/index.tsx]: Isn't that what this PR is doing?

Comment (josephspens) [reply src/ticketmap/index.tsx]: The goal of this PR is to exclude nodes _when they have children_. If a node doesn't have children, we include it, else we exclude it and include it's children.

Comment (josephspens) [reply src/ticketmap/index.tsx]: For example a `rect` element doesn't have children, if it has a `data-section-id` attribute then we want to include it. A `group` element does have children, we would want to include its children and exclude the `group` element.

Comment (mayavera) [reply src/ticketmap/index.tsx]: 👌

**Pull request merged**